Create My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md

srija-ieee opened the pull request:
Please review and publish this article

srija-ieee committed:
Create My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
```diff
@@ -0,0 +1,61 @@
+---
```
Rename the file as 2020-7-04-drsrija-intro-rfkill.md

srija-ieee replied:
Done

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
```diff
@@ -0,0 +1,61 @@
+---
+title: "My Introduction to Wi-Fi 'Hard block' and 'rfkill'"
+date: 2020-07-26T15:34:30-04:00
```
You have written in the future!

Please set the date correct. Its 26th July as per the timestamp

srija-ieee replied:
Changed it, thanks for pointing out !

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
```diff
@@ -0,0 +1,61 @@
+
+Surmising an issue in the Wi-Fi driver, I removed the exisiting Wi-Fi packages using  _'apt'_ & _'purge'_ tools and then re-installed them, but couldn't find any change!!
+
+I then checked the list of pci buses on my system using _'lspci'_ command and  realised  “Network Bus’’ was missing. 
```
Code high-lighting may be used. See the triple backtick ``` format
[Code highlighting in GFM](https://docs.github.com/en/enterprise/2.13/user/articles/creating-and-highlighting-code-blocks)

srija-ieee replied:
corrected

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
```diff
@@ -0,0 +1,61 @@
+
+When these devices are deactivated they fall into **'soft block'** ( a state in which the device can be re-activated using software) **or**  **'hard block'** ( a state in which the device cannot be re-activated using a software ) 
+
+( References:
```
See how references are written in GFM.

Basically you use [[ref1]]  or [[ref any name]] and then add the refs at the end as
```md
Use [ref1] [ref2] [any ref name] in the text, and...

...more text...

...more text with refs...

...rest of the text...

At the end of the document provide the references
[ref1]: https://ayeai.xyz
[ref any name]: https://pvjournal.github.io
```

[ref1]: https://ayeai.xyz
[ref any name]: https://pvjournal.github.io

srija-ieee replied:
done

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
```diff
@@ -0,0 +1,61 @@
+
+As mentioned in the beginning of my writeup,  I remembered I was adjusting the brightness using keyboard buttons (Function +F12) supposing this action might have blocked Wi-FI I tried it again but no luck...
+
+Then I tried '**Function + Prtscr**' ( ‘PrtScr’ has a transmittor symbol on it)  and finally it worked!!!
```
This is not a generic keyboard layout. You need to identify your device make and model. At least the available details must be provided, while being **careful not to reveal your personal details**

srija-ieee replied:
True, added mentioned  name and os of laptop at the beginning of the sentence

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
````diff
@@ -0,0 +1,61 @@
+ Looking back at my initial action I might accidentally stroked ‘PrtScr’ key (which is right beside F12) while holding 'Function (Fn)' key
+
+It is important to be vigilant of the actions and keyboard strokes you give while using the keyboard otherwise we may end up triggering changes in hardware setup!!!!
+```c
````
Remove the content below this. It comes from the sample text.

srija-ieee replied:
done

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
````diff
@@ -0,0 +1,61 @@
+}
+```
+
+Check out the [Jekyll docs][jekyll-docs] for more info on how to get the most out of Jekyll. File all bugs/feature requests at [Jekyll’s GitHub repo][jekyll-gh]. If you have questions, you can ask them on [Jekyll Talk][jekyll-talk].
````
Look at this example of how references are given

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
```diff
@@ -0,0 +1,61 @@
+Then I tried '**Function + Prtscr**' ( ‘PrtScr’ has a transmittor symbol on it)  and finally it worked!!!
+
+ Looking back at my initial action I might accidentally stroked ‘PrtScr’ key (which is right beside F12) while holding 'Function (Fn)' key
+
```
Remove everything after this - they are all from the sample

srija-ieee replied:
done

obonac commented on _posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md:
```diff
@@ -0,0 +1,61 @@
+[jekyll-gh]:      https://github.com/jekyll/jekyll
+[jekyll-talk]:    https://talk.jekyllrb.com/
+[pvjournal]:      https://pvjournal.github.io/
+[project-vikram]: https://projectvikram.github.io/
```
Provide a sample screen shot of of the post rendering in your fork of the journal / magazine / blog (in this case the journal)

srija-ieee replied:
I forked the repo and changed its name and the page appears on this link - https://drsrija-work.github.io/. In the '_posts'  there was 'test-post.md' file,I created a new file and copied the format from the test-post.md file and named as '2020-7-04-drsrija-intro-rfkill.md'. my posts doesn't show up on the page.. 
![posts pic](https://user-images.githubusercontent.com/42797168/86529644-c6245900-becf-11ea-8d7e-6ff64bf2078a.png)

srija-ieee replied:
Resolved: 

The changes have to be pulled into master branch to get rendered onto the website..
A pull request has to be raised from the journal branch to master branch and we need to merge it into the master branch. then the page shows up : 

![resolved posts](https://user-images.githubusercontent.com/42797168/86531555-f4aa3000-bedf-11ea-9e86-2941899f38b9.png)

srija-ieee committed:
Update and rename My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md to 2020-7-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update 2020-7-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update _config.yml

srija-ieee committed:
Rename 2020-7-04-drsrija-intro-rfkill.md to 2020-07-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update 2020-07-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update 2020-07-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update 2020-07-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update 2020-07-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update _config.yml

srija-ieee committed:
Update _config.yml

srija-ieee committed:
Update Gemfile

srija-ieee committed:
Update Gemfile

srija-ieee committed:
Merge pull request #1 from drsrija-work/journal

Test rendering

srija-ieee committed:
Update 2020-07-04-drsrija-intro-rfkill.md

srija-ieee committed:
Update 2020-07-04-drsrija-intro-rfkill.md

srija-ieee committed:
Added image

srija-ieee committed:
Added Image

srija-ieee committed:
Update team.md

srija-ieee committed:
Update team.md

srija-ieee committed:
Update team.md

srija-ieee committed:
Update team.md

srija-ieee committed:
Merge pull request #2 from drsrija-work/master

Merge pull request #1 from drsrija-work/journal

srija-ieee committed:
Update 2020-07-01-test-post.md

srija-ieee committed:
Create drsrija-hindawi-gcolab.md

srija-ieee committed:
Update drsrija-hindawi-gcolab.md

srija-ieee committed:
Update drsrija-hindawi-gcolab.md

srija-ieee committed:
Update drsrija-hindawi-gcolab.md

srija-ieee committed:
Add files via upload

srija-ieee committed:
Add files via upload

srija-ieee committed:
fixed spacing, puntuations, etc..

Ms Anamika's Pictured is not getting rendered on  the page.. the link has been taken from LinkedIN.. , Need to recheck and fix it

srija-ieee committed:
Update _config.yml

srija-ieee committed:
Create disclaimer for healthcare

srija-ieee committed:
Rename disclaimer for healthcare to disclaimer for healthcare.md

srija-ieee committed:
Update disclaimer for healthcare.md
